### ISSUE TITLE
Apple of Idun's Musical Lesson scaling

### ISSUE THREAD

**Issue by XanKriegor1** [rathena/rathena#7727]
### rAthena Hash

latest

### Client Date

2022-04-06

### Server Mode

Pre-Renewal

### Result

I'm going through the process of reviewing skill descriptions, comparing it between sources and the actual values in rAthena.
While going through Apple of Idun, I found a discrepancy.
iROwiki states that every level of Musical Lesson increases the MaxHP increase by 1%.
And on rAthena's status.cpp(line 12190) there's a comment that states:
//HP Rate: (5 + 2 * skill_lv) + (vit/10) + (BA_MUSICALLESSON level)
However, two lines beneath that, it has:
val2 += pc_checkskill(s_sd, BA_MUSICALLESSON) / 2;
In other words, you only get 1% HP every 2 skill levels.

### Expected Result

Descriptions should match implementation.

### How to Reproduce

Try using a bard in-game and trying the buff on another player, boosting musical lessons by 1 level at a time. You only get higher HP values every 2 skill levels.

### Official Information

Based on iROwiki classic description. The internal comment on rA has the value stated on iROwiki, but the code differs.

**Comment by Playtester** [comment]
Same as the other topics, please check https://github.com/rathena/rathena/issues/1808 for official values.
